test/data/tokenizer/test4.test

```diff
@@ -152,14 +152,6 @@
 "input":"&#4294967361;",
 "output":["ParseError", ["Character", "\uFFFD"]]},
 
-{"description":"65-bit hex numeric entity",
```

Comment:
+ @jmesserly - possibly Jenny has more insights into this, but I thought these tests were purposely covering >64 bit because it is technically a valid part of the html spec.

My guess is that rather than deleting the tests, we should mark them as failing, or create a new bug entry with the 2 tests so coverage can be added again in the future (likely by changing int.parse with BigInt.parse or something similar.)